backend/src/db/types/OAuth2.ts

```diff
@@ -0,0 +1,68 @@
+export interface OAuth2ServerAccessToken {
+  accessToken: string;
+  accessTokenExpiresAt: Date;
+  scope: string[];
+  client: {
+    id: string;
+  };
+  user: {
+    id: number;
+  }
+}
+
+export interface OAuth2ClientRaw {
+  id: number;
+  name: string;
+  description: string;
+  logo_url: string;
+  client_id: string;
+  client_secret_hash: string;
+  client_secret_original?: string;
+  initial_authorization_url: string;
+  redirect_uris: string;
+  grants: string;
+  user_id: number;
+  is_public: number;
+  is_my?: number;
+  is_authorized?: number;
+}
+
+export interface OAuth2Token {
+  access_token: string;
+  expires_in: number;
+  token_type: 'Bearer';
+  refresh_token?: string;
+  scope: string;
+}
+
+export interface OAuth2TokenRaw {
+  access_token_hash: string;
+  access_token_expires_at: Date;
+  refresh_token_hash: string;
+  client_id: number;
+  client_client_id: string,
+  user_id: number;
+  scope: string;
+  revoked: number;
+}
+
+export interface OAuth2AuthorizationCodeRaw {
+  client_id: number;
+  client_client_id: string;
```

Comment:
really?

Reply:
wym?

Reply:
very confusing and inconsistent naming. please see my comment at the bottom with some suggestions how to address.